releasenotes/notes/pipeline-resume-state-13ca79f53277e858.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    A snapshot of the last successful step is now saved when an error occurs during a Pipeline run. This allows you
+    to inspect the snapshot, potentially identify and fix the error, and later resume the pipeline from that point
+    onwards. Avoiding to re-run the entire pipeline from the start.
```

Comment:
I think I would make it clear in the release note that this feature is only for `Pipeline` and not `AsyncPipeline`